plc4j/integrations/opcua-server/src/main/java/org/apache/plc4x/java/opcuaserver/backend/Plc4xNamespace.java

```diff
@@ -1,11 +1,20 @@
 /*
- * Copyright (c) 2019 the Eclipse Milo Authors
```

Comment:
i am no expert on this license topic. But is this ok in that way? That we have the small Milo notice in the "Notice" file and change the license?

Reply:
Unfortunately not ... EPL is category B ... we can use (in form of use libaries) but we can't include that code. ... not even just a bit. If it's just a bit, it would be better to re-write it.

Reply:
The EPL headers for the Plc4xNamespace and OPCUAServer files were added in as I wasn't sure how much of the code reflected the Milo example.
I am comfortable that there is no code that needs to be licensed under EPL remaining.